x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/expression/function/scalar/string/StringFunctionProcessorTests.java

```diff
@@ -102,13 +106,16 @@ public void testUCase() {
         stringCharInputValidation(proc);
     }
     
-    public void testUCaseWithTRLocale() {
+    public void testUCaseWithAZTRLocale() {
         Locale.setDefault(Locale.forLanguageTag("tr"));
         StringProcessor proc = new StringProcessor(StringOperation.UCASE);
 
         // ES-SQL is not Locale sensitive (so far).
         // in Turkish locale, small letter "i" is uppercased to "I" with a dot above (unicode 130), otherwise in "i" (unicode 49)
         assertEquals("\u0049", proc.process("\u0069"));
+        
+        Locale.setDefault(Locale.forLanguageTag("az"));
+        assertEquals("\u0049", proc.process("\u0069"));
```

Comment:
I think it might makes sense to restore the original locale after the test so other tests in this class don't run in this local most of the time.

Reply:
Same deal here with restoring in a finally